app/models/contributor.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
I'd like to discuss what to do here @phoeinx @tillprochaska @roschaefer ... I don't like just disabling this, but addressing the issue `rubocop` brings up seems outside of the scope of this `PR`.

Also, one thing I don't like about our pre-push hooks is that I can't even push up the code unless I address linting issues. If it is just a matter of running `rubocop -a` then fine, but when they require more thought and action, I don't want to leave code un-pushed and I don't want to be late to dinner or work longer than I have for a day so that I can push my code.

Reply:
> Also, one thing I don't like about our pre-push hooks is that I can't even push up the code unless I address linting issues. If it is just a matter of running rubocop -a then fine, but when they require more thought and action, I don't want to leave code un-pushed and I don't want to be late to dinner or work longer than I have for a day so that I can push my code.

I personally have don’t have the git hooks set up because I find them annoying for exactly the same reason and suggested removing them some time ago. But I think Robert (?) that he actually finds them useful?

So if you’re annoyed by them, maybe simply disable/remove them for now?

Reply:
That would be a solution. I don't know how to disable them. I quickly searched for it after I read your comment and didn't find anything.

I would personally be in favor of removing them for the reasons stated. The downside would be that we would use more minutes if we push up code that fails linting in our actions.

Reply:
I personally like the hooks exactly because I otherwise tend to forget linting the code and it then fails in our actions, which means I have to refocus on the code again even though I probably started something new already.

Reply:
> That would be a solution. I don't know how to disable them. I quickly searched for it after I read your comment and didn't find anything.

You can run `rm -rf ./.git/hooks` to remove all installed Git hooks. I think you might need to do that again when you (re-)install lefthook from npm.

Reply:
No one commented on the fact that we just disable the cop. I guess we might want to see if there is a way we can make this model skinnier?

This would be another ticket then, I believe. Or, are people not concerned with this cop in this case?

Reply:
I would like to, to be honest cause I am interested in this type of refactoring, I just don't know if it provides much business value and so don't know if it can be prioritized. Just didn't want to not mention the fact that I am just disabling the cop so that we don't have to address the underlying issue.